### ISSUE TITLE
[UX] There is no indication that once a Teacher, Student, Parent is created - a new user should be created and associated with it

### ISSUE THREAD

**Issue by anjanamohanty** [tiyd-rails-2016-01/grade_master#56]

**Comment by stevengcooper** [comment]
I was just looking at that.  Do we need a create new user in that list?  We create teachers, parents, and students in their manage page.

**Comment by anjanamohanty** [comment]
You can create a teacher, student, parent independently -- but they won't be able to have a login until you create a new user and associate them with that new created teacher, student, parent

**Comment by stevengcooper** [comment]
got you.  That is an issue.

**Comment by anjanamohanty** [comment]
I don't believe its as high priority as everything else, though. Its technically still shippable.

**Comment by Publicover** [comment]
every job i've ever had used some kind of central file system. i was granted a login and a password, and i could change the password at my leisure. but i sure couldn't create my own login credentials since i wasn't an admin. 

this isn't too far from real-world conditions, is what i'm saying. i vote to icebox this one.
